src/ui/Button.tsx

```diff
@@ -224,7 +224,7 @@ const customCSS = css`
       background-color: ${getValue('icon.hoverBackgroundColor')};
     }
 
-    &:focus {
+    &:active {
```

Comment:
Maybe you can add the active state instead of replacing the :focus state?
Unless the focus state is not necessary?

```suggestion
    &:active,
    &:focus {
```

Reply:
I added the focus state by mistake; what I actually meant to add was the active state.